src/sentry/integrations/slack/tasks/find_channel_id_for_alert_rule.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.integrations.slack.find_channel_id_for_alert_rule import (
+    find_channel_id_for_alert_rule as old_find_channel_id_for_alert_rule,
+)
+
+
+@instrumented_task(
+    name="sentry.integrations.slack.tasks.search_channel_id_metric_alerts",
```

Comment:
```suggestion
    name="sentry.integrations.slack.tasks.search_channel_id_for_alert_rule",
```

We can make the task names match the function/module path now.